ramses/acl.py

```diff
@@ -208,13 +231,18 @@ def generate_acl(model_cls, raml_resource, es_based=True):
             acl_string=settings.get('item'),
             methods_map=item_methods)
 
-    class GeneratedACL(BaseACL):
+    class GeneratedACLBase(object):
```

Comment:
Should this inherit from `nefertari.acl.CollectionACL`? It's a little unclear to mean how this is used and whether a bunch of other classes are mixed in later or not.